docs/cont/contributing.md

```diff
@@ -102,13 +102,13 @@ If you decide to use a different license for your work, follow the same pattern:
 
 * Begin each source header with your copyright line, the SPDX identifier and the standard header for the license that applies to that single file, if it has one.
 
-* If more than one license applies to the source file, then use an SPDX license expression (see Appendix IV in [this document](http://spdx.org/sites/spdx/files/SPDX-2.0.pdf)), to reflect the presence of multiple licenses in your *LICENSE* file and in each source file.
+* If more than one license applies to the source file, then use an SPDX license expression (see Appendix IV in [this document](https://spdx.org/spdx_specification_2_0_html)), to reflect the presence of multiple licenses in your *LICENSE* file and in each source file.
```

Comment:
You're *sure* about the URL? You should check it with someone who knows about contribution and legal.